tailwind.config.js

```diff
@@ -12,6 +16,6 @@ module.exports = {
         para: '40ch',
         half: '50%'
       }
-    }
-  }
+      }
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8426368)
```suggestion
    }
```